src/i_video.c

```diff
@@ -1779,6 +1786,43 @@ void I_InitGraphics(void)
     I_ResetRelativeMouseState();
 }
 
+void I_BindVideoVariables(void)
+{
+    BIND2_NUM(current_video_height, 600, SCREENHEIGHT, UL,
+        "Vertical resolution (600p by default)");
+    BIND_BOOL_GEN(dynamic_resolution, true, "1 to enable dynamic resolution");
+    BIND_BOOL(correct_aspect_ratio, true, "1 to perform aspect ratio correction");
+    BIND_BOOL(fullscreen, true, "1 to enable fullscreen");
+    BIND_BOOL(exclusive_fullscreen, false, "1 to enable exclusive fullscreen");
+    BIND_BOOL_GEN(use_vsync, true,
+        "1 to enable wait for vsync to avoid display tearing");
+    BIND2_BOOL_GEN(uncapped, true, "1 to enable uncapped rendering frame rate");
+    BIND_NUM_GEN(fpslimit, 0, 0, 500,
+        "Framerate limit in frames per second (< 35 = disable)");
+    BIND2_NUM_GEN(widescreen, RATIO_AUTO, 0, NUM_RATIOS - 1,
+        "Widescreen (0 = Off, 1 = Auto, 2 = 16:10, 3 = 16:9, 4 = 21:9)");
+    M_BindNum("fov", &custom_fov, NULL, FOV_DEFAULT, FOV_MIN, FOV_MAX,
```

Comment:
The changes in `M_BindNum()`, `BIND_NUM_GEN()`, `BIND2_NUM()`, etc. are confusing.

Reply:
So should I get rid of these macros? I think we can keep the most useful ones `BIND_NUM`, `BIND_BOOL`, `BIND_NUM_GEN` and `BIND_BOOL_GEN`. (`GEN` refers to the "General" options menu).

Reply:
Yes, probably. It doesn't help to have that many macros with similar names but rather specific uses. Also, could you rename `GEN` to `GENERAL`?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1677/commits/c00fcc35c51b941a95111df3372ef0334cc2f58c